app/views/header.js

```diff
@@ -0,0 +1,36 @@
+import Mn from 'backbone.marionette';
+import application from '../application';
+import Tracks from '../collections/tracks';
+
+const Header = Mn.ItemView.extend({
+
+    template: require('./templates/header'),
+
+    events: {
+        'click #reset-upnext': 'resetUpNext',
+        'click #delete-playlist': 'deletePlaylist'
+    },
+
+    //modelEvents: { change: 'render' }, // DON'T WORK
+
+    initialize: function() {
+        this.model = application.headerInfos;
+        this.listenTo(this.model, 'change', this.render)
+    },
+
+    serializeData: function() {
+        return {
+            title: this.model.get('title'),
+            count: this.model.get('count')
+        }
+    },
+
+    resetUpNext: function() {
+        application.upNext.setAttr('currentTrack', undefined);
+        application.upNext.reset();
+        application.appLayout.getRegion('player').currentView.render();
```

Comment:
Glitch: you should not be able to call another view directly. Instead, pass an event and catch it in the player view.